app/views/home/index.html.erb

```diff
@@ -2,29 +2,46 @@
    HEADERWRAP
    ***************************************************************************************************************** -->
 <!-- Header -->
-<section class="jumbotron span-12 one-edge-shadow">
-  <div class="container">
-    <div class="row">
-      <h1>Ask<span>Up</span></h1>
-      <h3>Learn better, together.</h3>
-    </div><br><br>
-    <div class="row">
-      <div class="col-sm-4  center">
-        <div class="row"><%= link_to "Generate Question", new_question_path, class: "btn btn-theme btn-lg" %></div>
 
+    <div class="row jumbotron one-edge-shadow">
+        <div class="container-row">
+          <div class="cell">
+            <h1>Ask<span>Up</span></h1>
+            <h3>Learn better, together.</h3>
+          </div>
+          <div class="question-cell">
+              <div class="center"><%= link_to "Generate Question", new_question_path, class: "btn btn-theme btn-lg" %></div>
+              <!--<div class="center">-->
```

Comment:
If you've tested the app and you know it's working, it's a good idea to remove all these comments for lines / elements we no longer need.